### ISSUE TITLE
[BUG] docker-compose-postorius.yaml is unmaintained

### ISSUE THREAD

**Issue by duck-rh** [maxking/docker-mailman#605]
**Tag/version of Container Images**
0.4.5

Quack,

First I would like to point out that the doc does not make it clear that `docker-compose-postorius.yaml` is postorius-only while the other one is postorius+hyperkitty and in fact I chose to use this one to be sure to also get postorius installed, fail.

Anyway, I noticed the example PostgreSQL links use `postgres://` which is deprecated, but `docker-compose.yaml` was updated to the `postgresql://` URL.

Also recently sqlalchemy was bumped and the PostgreSQL container too since this verison requires at least PG 11, but `docker-compose-postorius.yaml` as not updated.

I think that's all but I might have missed other differences.

I'll get back to trying to make this work. Thanks for maintaining these images.
Regards.
\\_o<

**Comment by maxking** [comment]
> First I would like to point out that the doc does not make it clear that docker-compose-postorius.yaml is postorius-only while the other one is postorius+hyperkitty and in fact I chose to use this one to be sure to also get postorius installed, fail.

Yeah, we are missing specific docs related to the postorious related image.


Also, there is a lack of testing in general for the postorius image and that’s something i’ve been meaning to get to. I’ll keep this issue open to get it to it. It doesn’t get used a lot either (atleast, based on the questions I get) and for how long it remains broken before someone reports it.

I’ll try to extend the basic testing we do for other 2 container images with Postorius.

**Comment by beardedfool** [comment]
Slight embarrased, as a noob, with how long I spend trying to get that work.

Could you maybe drop a note in compose-postorius.yaml with a reference to this ticket please, for the next guy?

Cheers for all you've done here

**Comment by jakicoll** [comment]
Just a quick note: #650 fixes this, since the String "postgres" in the docker-compose-postorious.yaml will be replaced by  $DATABASE_URL_SCHEMA, which is set to "postgresql" via the .env file.
